presto-parser/src/main/antlr4/com/facebook/presto/sql/parser/SqlBase.g4

```diff
@@ -101,7 +101,7 @@ tableElement
     ;
 
 columnDefinition
-    : identifier type (COMMENT STRING)?
```

Comment:
I'm still using the current structure instead of creating two new rules given that a escaped string needs to have 'U&' to be stripped out first then unquote. In the end. Seems there isn't much difference.